fallback_test.go

```diff
@@ -1,255 +0,0 @@
-package fallback
```

Comment:
What about the UT for alternate ?

Reply:
See comment above:
Also, there is no alternate main UT now. But it is pretty tricky. I will think of it and I will implement it tomorrow if it takes sensible time.

Reply:
ok